sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentTableCellKind.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for DocumentTableCellKind. */
 @Immutable
 public final class DocumentTableCellKind extends ExpandableStringEnum<DocumentTableCellKind> {
+
+    /**
+     * Creates a DocumentTableCellKind object.
+     */
+    public DocumentTableCellKind() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.